build.sbt

```diff
@@ -90,8 +90,10 @@ lazy val cats = crossProject(JSPlatform, JVMPlatform)
     name := "paiges-cats",
     moduleName := "paiges-cats",
     libraryDependencies ++= Seq(
-      "org.typelevel" %%% "cats-core" % "1.6.1",
-      "org.typelevel" %%% "cats-laws" % "1.6.1" % Test),
+      "org.typelevel" %%% "cats-core" % "2.0.0-M4",
+      "org.typelevel" %%% "cats-laws" % "2.0.0-M4" % Test,
+      "org.typelevel" %%% "discipline-core" % "0.12.0-M3" % Test,
```

Comment:
it's used by cats-laws, so you don't really need this.

Reply:
Removed, thank you